recipe/meta.yaml

```diff
@@ -1,27 +1,27 @@
 {% set name = "aws-c-cal" %}
-{% set version = "0.6.9" %}
+{% set version = "0.5.20" %}
```

Comment:
Can you remind me why we are sticking to older versions?

Reply:
1. https://github.com/aws/aws-sdk-cpp/tree/1.10.55/crt - this is 0.18.16
2. https://github.com/awslabs/aws-c-cal/tree/ac4216b78d5323b5b8ce95a3dd4a8fc0f95e2d33 - this is 0.5.20